libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -333,8 +350,20 @@ bool AP_Arming::battery_checks(bool report)
             }
             return false;
         }
-    }
 
+        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
```

Comment:
Lol no that part is obvious :P I meant the parameter table on the
autopilot. (IE increasing the array size via increasing the constant, but
forgetting to add a new param, thus leading to checking an undefined value
from the constant table against the reported voltage.)

On Wed, Mar 2, 2016 at 12:45 PM, Tom Pittenger <notifications@github.com>
wrote:

> In libraries/AP_Arming/AP_Arming.cpp
> <https://github.com/diydrones/ardupilot/pull/3604#discussion_r54779295>:
>
> >
> > +        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
> > +            if (report) {
> > +                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
>
> do you mean in the print? Just use:
> GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery
> %d voltage below minimum of %.1fV", index, _min_voltage[index]);
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3604/files#r54779295>.
>